src/tag/tag.scss

```diff
@@ -1,83 +1,86 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
-@import '../icon/index';
 @import '../text/mixins';
+@import '../button/index';
 
-@mixin iui-tag {
+@mixin iui-tag-base {
   @include iui-reset;
   @include iui-focus($offset: -2px, $thickness: 2px);
   user-select: all;
   text-transform: lowercase;
   display: inline-flex;
+}
+
+@mixin iui-tag-basic {
+  @include iui-tag-base;
+  outline-offset: 1px;
```

Comment:
You have `outline-offset` set here in the CSS and you are setting it when including the `iui-focus` mixin within `classes.scss`.  Are both required?

Reply:
I guess not.. removed